charts/opentelemetry-collector/UPGRADING.md

```diff
@@ -16,6 +16,34 @@ created but the Memory Ballast Extension will still be removed.
 Depending on the progress made in [Issue 891](https://github.com/open-telemetry/opentelemetry-helm-charts/issues/891),
 the use of `GOMEMLIMIT` may completely replace the Memory Ballast Extension in the future.
 
+## 0.87.2 to 0.88.0
+
+As part of working towards using the [OpenTelemetry Collector Kubernetes Distro](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-k8s) by default, the chart now requires users to explicitly set an image repository and command name. If you are already explicitly setting an image repository and command name this breaking change does not affect you.
+
+If you are using a OpenTelemetry Community distribution of the Collector we recommend you use `otel/opentelemetry-collector-k8s`, but carefully review the [components included in this distribution](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-k8s/manifest.yaml) to make sure it includes all the components you use in your configuration. In the future this distribution will become the default image used for the chart.
```

Comment:
why are we looking to move away from the contrib distribution in the helm chart?

Reply:
contrib really isn't meant to be run in a production setting ([doc](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib#recommendation)). Contrib contains every component with stability greater than or equal to alpha, which means it probably has a lot of stuff the average user doesn't need. This opens up a potential attack vector if there's a vuln in one of said components that users are unknowingly pulling in. It also means that the overall image size is much larger than you may need which would result in worse first-pull latency. There are a few other reasons that are documented around, but these are the two biggest.